Facade_Engine/Query/PrimaryPropertyName.cs

```diff
@@ -71,7 +71,7 @@ public static string IPrimaryPropertyName(this IElement1D elem)
         [Output("propName", "The name of the element's construction property")]
         public static string PrimaryPropertyName(this FrameEdge frameEdge)
         {
-            if (frameEdge.FrameEdgeProperty == null)
+            if (frameEdge == null || frameEdge.FrameEdgeProperty == null)
                 return "";
```

Comment:
Can we provide a user warning here at a minimum to inform when there are nulls? Could get confusing to see an empty string if you think you provided all inputs correctly.